nwbwidgets/controllers/time_window_controllers.py

```diff
@@ -1,101 +1,102 @@
-from abc import abstractmethod
+from typing import Tuple, Optional
 
-from ipywidgets import HBox, Layout, ValueWidget, link, widgets
+from ipywidgets import Layout, ValueWidget, link, widgets
 from ipywidgets.widgets.widget_description import DescriptionWidget
 
+from .basecontroller import BaseController
 
-class WindowController(HBox, ValueWidget, DescriptionWidget):
+
+class WindowController(BaseController, ValueWidget, DescriptionWidget):
     def __init__(
-        self, vmin, vmax, start_value=(None, None), description="window (s)", orientation="horizontal", **kwargs
+        self,
+        vmin: int,
+        vmax: int,
+        start_value: Optional[Tuple[Optional[int], Optional[int]]] = (None, None),
```

Comment:
Not so sure about this type annotation; this class is the only one that initializes it with `(None, None)`, the others take `None` and figure out tuple contents from other values